src/Component/Symbolizer/Field/ImageField/ImageField.tsx

```diff
@@ -38,10 +37,18 @@ import { IconLibrary } from '../../IconSelector/IconSelector';
 
 import './ImageField.less';
 import { PictureOutlined } from '@ant-design/icons';
+import { Expression, IconSymbolizer, Sprite, isSprite } from 'geostyler-style';
+import StringExpressionInput from '../../../ExpressionInput/StringExpressionInput/StringExpressionInput';
+
+import './ImageField.less';
+import { FieldSet } from '../../../FieldSet/FieldSet';
+import NumberExpressionInput from '../../../ExpressionInput/NumberExpressionInput/NumberExpressionInput';
 
 export interface ImageFieldProps {
   /** The tooltip label for the iconLibraries button. */
   tooltipLabel?: string;
+  /** The tooltip label for the sprite button. */
+  spriteTooltipLabel?: string;
```

Comment:
Could you add the `spriteTooltipLabel` to the i18n files?

Reply:
Solved with [5b68e8d](https://github.com/geostyler/geostyler/pull/2389/commits/5b68e8d8bbb88610da38d42f5e02083119e0c0fc)